src/app/CommandHandler.cpp

```diff
@@ -746,11 +766,31 @@ CHIP_ERROR CommandHandler::FinishStatus()
     return CHIP_NO_ERROR;
 }
 
+void CommandHandler::CreateBackupForResponseRollback()
+{
+    if (mState != State::NewResponseMessage && mState != State::AddedCommand)
+    {
+        return;
+    }
+    if (mInvokeResponseBuilder.GetInvokeResponses().GetError() != CHIP_NO_ERROR ||
+        mInvokeResponseBuilder.GetError() != CHIP_NO_ERROR)
+    {
+        return;
+    }
```

Comment:
These 3 conditions can be converted to 3 separate `VerifyOrReturn`. The second if() with || could be split into 2 separate VerifyOrReturn.